admin_guide/manage_authorization_policy.adoc

```diff
@@ -565,3 +565,96 @@ administrator, you can grant them the abilities.
 ----
 $ oadm policy add-role-to-user system:daemonset-admin <user>
 ----
+
+[[manage-authorization-policy-creating-local-role]]
+== Creating a Local Role
+
+To create a local role for a project, it is recommended that you build it from
+scratch rather than copying an existing role.
+
+.Example Role
+----
+apiVersion: v1
+kind: Role
+metadata:
+  name: role_name
+rules:
+- apiGroups: null
+  attributeRestrictions:
+    apiVersion: v1
+    kind: IsPersonalSubjectAccessReview
+  resources:
+  - localsubjectaccessreviews
+  - subjectaccessreviews
+  verbs:
+  - create
+----
+
+.Full Go Specification
+----
+// Role is a logical grouping of PolicyRules that can be referenced as a unit by RoleBindings.
```

Comment:
Currently OOO. But yes, I never intended for the source code to be included. It was mostly there as a reference so that whatever YAML we included was thorough and correct.